broker/src/main/java/io/zeebe/broker/transport/backpressure/CommandRateLimiter.java

```diff
@@ -65,7 +69,13 @@ public boolean tryAcquire(final int streamId, final long requestId, final Intent
   public void onResponse(final int streamId, final long requestId) {
     final Listener listener = responseListeners.remove(new ListenerId(streamId, requestId));
     if (listener != null) {
-      listener.onSuccess();
+      try {
+        listener.onSuccess();
+      } catch (final IllegalArgumentException e) {
+        LOG.warn("Ignoring request in command rate limiter due to: ", e);
```

Comment:
I'm wondering if we shouldn't recommend something to the user here, or maybe in the docs, as I can imagine users running into this wouldn't be sure what to do.

Reply:
Any suggestions? Maybe it's worth stating repeatedly ignore the request will make the request limiter less accurate. It would be easier if we could there was a specific cause we could point to, it would be more actionable

Reply:
We could direct users to our reference documentation, and in there we can explain the situation in more details.

Reply:
Sounds good